### DISCUSSION TITLE
Discussion: Advertising networks for WinUI 3 apps

### DISCUSSION THREAD

**Discussion by Sergtek** [microsoft/microsoft-ui-xaml#7824] [Ideas]
I'd like to discuss the possibility of the WinUI team working with ad network partners to support apps that use WinUI 3 so they can be easily monetized with ads. In UWP we have several providers such as [Vungle ](https://vungle.com/)or [AdsJumbo](https://adsjumbo.com/), I think that the fact of having platforms of this type will help in the adoption of WinUI 3 by many companies and people who live from the profits produced by the ads in their applications. What do you think about it?

**Comment by fetisenko** [comment]
It was nice to have Microsoft Ads on UWP a few years ago. All ads formats supported, including my favorite Native Ads. Decent income.

Shame that Microsoft is not Windows Dev friendly anymore.

**Comment by DamianSuess** [comment]
Yes, as a content creator i want to publish to the store and make it free to users, however, having Ad revenue would be beneficial to help pay for developer costs.

Really wish AdMob was usable for MAUI or WinUI, or some other reliable source. Any word on this?

**Comment by Khaos66** [comment]
There is something coming up
https://learn.microsoft.com/en-us/windows/apps/publish/in-app-ads

**Reply by moontain-dev** [reply]
Seems nothing has come up, the link was redirected to another page...

**Reply by Khaos66** [reply]
Bummer 😞 was hoping to monetize my app... I'll add a 'Here could be ads' screen and make the user buy a license to hide it 🥴

**Comment by DilanBoskan** [comment]
The year is coming to an end

**Reply by ereinaimer** [reply]
have you found any native support networks yet?

**Comment by ereinaimer** [comment]
has anyone found any ad network or workarounds for this? I'm really hoping some ad network had winui3 sdk or support of any kind. 

if you do, kindly reach out to me.

**Comment by idenardi** [comment]
I haven't tested yet, but it seems possible with Litoff - [Integrate Vungle SDK for Win32, WinUI 3, or WinForms
](https://support.vungle.com/hc/en-us/articles/45026606409371-Integrate-Vungle-SDK-for-Win32-WinUI-3-or-WinForms#h_01M2NGQ2VC2BMEH0P3FPB896NP)
